src/java/main/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -273,7 +275,7 @@ protected void connectToZK(String newHost) throws InterruptedException, IOExcept
         host = newHost;
         boolean readOnly = cl.getOption("readonly") != null;
         if (cl.getOption("secure") != null) {
-            System.setProperty(ZooKeeper.SECURE_CLIENT, "true");
+            System.setProperty(ZKClientConfig.SECURE_CLIENT, "true");
```

Comment:
For backward compatibility, I think `ZKClientConfig.SECURE_CLIENT` must be equal to `ZooKeeper.SECURE_CLIENT`. Until we remove the deprecated properties, we should perhaps make ` ZKClientConfig.SECURE_CLIENT = ZooKeeper.SECURE_CLIENT`.